### PR TITLE
Change logging procedure

### PR DESCRIPTION
**Pull request by l-vincent-l**
We now only use datagouv oauth, so we don't need the auth0 lock.

Fixes etalab/transport-datatools-server#12

### CODE REVIEW

File: lib/common/user/Auth0Manager.js [hunk 1]
```diff
@@ -12,7 +13,26 @@ const icon = '' // import icon from '../../assets/application_logo.png'
 export default class Auth0Manager {
   constructor (props) {
     this.props = props
-    this.lock = new Auth0Lock(process.env.AUTH0_CLIENT_ID, process.env.AUTH0_DOMAIN)
+    this.webauth = new auth0.WebAuth({
+        clientID: process.env.AUTH0_CLIENT_ID,
+        domain: process.env.AUTH0_DOMAIN,
+        redirectUri: "http://localhost:9966",
```
Comment (maukoquiroga) [inline lib/common/user/Auth0Manager.js]: Would that work in prod ?

File: lib/common/user/Auth0Manager.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
-import Auth0Lock from 'auth0-lock'
+import * as auth0 from 'auth0-js'
 import fetch from 'isomorphic-fetch'
 import { browserHistory } from 'react-router'
+import promisify from 'es6-promisify'
```
Comment (bonjourmauko) [inline lib/common/user/Auth0Manager.js]: Why ?

File: lib/common/user/Auth0Manager.js (hunk 1 above)
Comment (bonjourmauko) [inline lib/common/user/Auth0Manager.js]: I don't think that'd work in production.

**Review by bonjourmauko — CHANGES_REQUESTED** [review]
Not production ready yet.

**Comment by bonjourmauko** [comment]
@l-vincent-l You tell me when it's ready for another review.

**Comment by l-vincent-l** [comment]
depends on https://github.com/etalab/transport-datatools-server/pull/16

File: lib/common/user/Auth0Manager.js [hunk 3]
```diff
@@ -22,22 +21,25 @@ export default class Auth0Manager {
...
             scope: "openid",
             redirectUri: window.location.href,
             responseType: "token",
         }, callback)
   }
 
-  isTokenFresh() {
-      return JSON.stringify(new Date()) < this.getExpireDate()
+  isRenewNeeded() {
+      return !!this.getToken() && this.isTokenExpired();
```
Comment (bonjourmauko) [inline lib/common/user/Auth0Manager.js]: Why are you using double negation ? For casting ?

**Review by bonjourmauko — APPROVED** [review]

**Pull request merged**